docs/getting-started.md

```diff
@@ -139,5 +139,6 @@ public class BusService : BackgroundService
 ```
 
 * Note here that we're calling `StartAsync` on both the `IMessagingBus`, and `IMessagePublisher`. Starting the `IMessagePublisher` ensures that all publish infrastructure is created and ready, while starting the `IMessagingBus` will firstly ensure that all subscription infrastructure is created and ready, and then begin consuming messages.
-* The only way to stop the bus is to cancel the cancellation token, which will tear down everything.
+* The only way to stop the bus is to cancel the cancellation token, which will tear down everything; however you can effectively pause listening on the bus by
+using the methods on a singleton instance of `MessageReceiveController` to control the receiving of all messages, even while the bus is running.
```

Comment:
This should point the user to the interface in DI, rather than the concrete type. Otherwise if the implementation were changed in DI, using the concrete type wouldn't do anything because the instance actually being used by the internals would be something else.